File: sdk/AMQP/python/one-to-one/producer.py
# File: producer.py
# Description: AMQP protocol. This is consumer code which can get message from exchange and consume them. One-to-one method.
# Author: Stanley
# robomq.io (http://www.robomq.io)
import pika

connection = pika.BlockingConnection(pika.ConnectionParameters(
        host='your host'))
channel = connection.channel()

channel.queue_declare(queue='queueName')

channel.basic_publish(exchange='',
                      routing_key='queueName',
                      body='Hello World!')
print 'Sent 'Hello World!''
connection.close()

File: sdk/AMQP/nodeJS/one-to-one/producer.js
/**
* File: producer.js
* Description: AMQP protocol. This is consumer code which can get message from exchange and consume them. One-to-one method.
*
* Author: Stanley
* robomq.io (http://www.robomq.io)
*
*/
var amqp = require('amqp');
var connection = amqp.createConnection({ host: 'your host', port: 'port' });

connection.on('ready',function(){
	connection.exchange('', options={type:'direct',	autoDelete:false}, function(exchange){
		exchange.publish('routingKey', 'hello world');
        print('message sent');
	});
});
